src/components/node-list/node-list-row.js

```diff
@@ -124,7 +131,9 @@ const NodeListRow = memo(
                 'pipeline-nodelist__row__label--disabled': disabled,
               }
             )}
-            dangerouslySetInnerHTML={{ __html: label }}
+            dangerouslySetInnerHTML={{
+              __html: replaceMatches(label, stringsToReplace),
```

Comment:
Replace the `<>` here so we can still use `dangerouslySetInnerHTML` without a problem, specifically when searching.